app_dart/lib/src/request_handlers/scheduler/batch_backfiller.dart

```diff
@@ -92,7 +92,7 @@ class BatchBackfiller extends RequestHandler {
       });
       // Schedule all builds asynchronously.
       // Schedule after db updates to avoid duplicate scheduling when db update fails.
-      await Future.wait<void>(backfillRequestList(backfill), eagerError: true);
+      await Future.wait<void>(backfillRequestList(backfill));
```

Comment:
@christopherfujino is much more familiar with better handling of future errors, but I think you want something like:

```dart
final errors = [];
await Future.wait<void>(backfillRequestList(backfill).onError((e) => errors.add(e));
if (errors.isNotEmpty) {
  log.severe(errors);
}
```

Your original approach will consume all errorrs, and the catch will never be run.

Reply:
Based on doc, only the first error will be returned and it should trigger the catch?
```
If any future completes with an error, then the returned future completes with that error. If further futures also complete with errors, those errors are discarded.
```
But it would be good to catch all errors.

Reply:
I think the issue here is, if we're not `await`-ing `backfillRequestList`, will we even hit the `catch` clause? Let me try to test this out to confirm...

Reply:
Tested this with:

```dart
Future<void> main() async {
  try {
    await Future.wait(_getFutures());
  } catch (err) {
    print('caught $err');
  }
  print('done');
}

List<Future<void>> _getFutures() => <Future<void>>[
  Future<void>.delayed(const Duration(milliseconds: 40), () => throw Exception('error 1')),
  Future<void>.delayed(const Duration(milliseconds: 80), () => throw Exception('error 2')),
  Future<void>.delayed(const Duration(milliseconds: 300), () => print('finished third future')),
];
```

And it logged the expected:

```
finished third future
caught Exception: error 1
done
```

So this LGTM

Reply:
And good catch! This was a subtle bug.

Reply:
Thanks for the validation @christopherfujino !